mobile-app/app/screens/AppNavigator/screens/Loans/screens/CreateVaultScreen.tsx

```diff
@@ -0,0 +1,220 @@
+import { View } from '@components'
+import { Button } from '@components/Button'
+import { ThemedIcon, ThemedScrollView, ThemedSectionTitle, ThemedText, ThemedTouchableOpacity, ThemedView } from '@components/themed'
+import { StackScreenProps } from '@react-navigation/stack'
+import { useWhaleApiClient } from '@shared-contexts/WhaleContext'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import React, { useEffect, useState } from 'react'
+import NumberFormat from 'react-number-format'
+import { LoanParamList, LoanScheme } from '../LoansNavigator'
+import BigNumber from 'bignumber.js'
+import { useLogger } from '@shared-contexts/NativeLoggingProvider'
+
+type Props = StackScreenProps<LoanParamList, 'CreateVaultScreen'>
+
+export function CreateVaultScreen ({ navigation, route }: Props): JSX.Element {
+  const loanSchemes: LoanScheme[] = [
+    {
+      id: '1',
+      minColRatio: '150',
+      interestRate: '5'
+    },
+    {
+      id: '2',
+      minColRatio: '175',
+      interestRate: '3'
+    },
+    {
+      id: '3',
+      minColRatio: '200',
+      interestRate: '2'
+    },
+    {
+      id: '4',
+      minColRatio: '350',
+      interestRate: '15'
+    },
+    {
+      id: '5',
+      minColRatio: '500',
+      interestRate: '1'
+    },
+    {
+      id: '6',
+      minColRatio: '1000',
+      interestRate: '0.5'
+    },
+    {
+      id: '7',
+      minColRatio: '1000',
+      interestRate: '0.5'
+    },
+    {
+      id: '8',
+      minColRatio: '1000',
+      interestRate: '0.5'
+    },
+    {
+      id: '9',
+      minColRatio: '1000',
+      interestRate: '0.5'
+    },
+    {
+      id: '10',
+      minColRatio: '1000',
+      interestRate: '0.5'
+    }
+  ]
+  const client = useWhaleApiClient()
+  const logger = useLogger()
+  const [fee, setFee] = useState<BigNumber>(new BigNumber(0.0001))
+  const [selectedLoanScheme, setSelectedLoanScheme] = useState<LoanScheme | undefined>(route.params?.loanScheme)
+  const onSubmit = (): void => {
+    if (selectedLoanScheme === undefined) {
+      return
+    }
+
+    navigation.navigate({
+      name: 'ConfirmCreateVaultScreen',
+      params: {
+        loanScheme: selectedLoanScheme,
+        fee: fee
+      }
+    })
+  }
+
+  useEffect(() => {
+    client.fee.estimate()
+      .then((f) => setFee(new BigNumber(f)))
+      .catch(logger.error)
+  }, [])
+
+  return (
+    <ThemedScrollView
```

Comment:
too much padding on the bottom

Reply:
i think you meant the spacing below the button?
Updated, let me know if it's other spacing